client/components/login/LoginForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-sheet tile elevation="0" color="white">
+    <v-container class="py-16">
+      <v-row>
+        <v-col cols="6" class="mx-auto">
+          <h2 style="text-align: center; margin-bottom: 1rem">Login</h2>
+          <v-form ref="form" v-model="valid" lazy-validation>
+            <v-text-field
+              v-model="loginID"
+              clearable
+              outlined
+              solo
+              :rules="IDRules"
+              required
+              label="Login ID"
+            >
+            </v-text-field>
+            <v-text-field
+              v-model="password"
+              clearable
+              outlined
+              solo
+              label="Password"
+              :append-icon="show1 ? 'mdi-eye' : 'mdi-eye-off'"
+              :rules="PWRules"
+              :type="show1 ? 'text' : 'password'"
+              @click:append="show1 = !show1"
+            >
+            </v-text-field>
+            <v-btn
+              :disabled="!valid"
+              color="primary"
+              class="mr-4"
+              @click="validate"
+            >
+              Login
+            </v-btn>
```

Comment:
For forms, try not to disable a button if the field is invalid. Instead, check on submit of the form and give feedback where necessary. The reason for this is that the user may not know what is wrong with the form and feedback on submit is better for the user to know if they made a mistake.

```suggestion
            <v-btn type="submit" color="primary" class="mr-4">
              Login
            </v-btn>
```